concurrency-limits-servlet-jakarta/src/test/java/com/netflix/concurrency/limits/ConcurrencyLimitServletFilterTest.java

```diff
@@ -14,18 +15,25 @@
 import jakarta.servlet.http.HttpServletRequest;
 import java.io.IOException;
 import java.util.Optional;
+import java.util.function.Predicate;
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertNull;
 import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.isA;
+import static org.mockito.Mockito.doReturn;
+import static org.mockito.Mockito.times;
 import static org.mockito.Mockito.verify;
-import static org.mockito.Mockito.when;
 
 @ExtendWith(MockitoExtension.class)
 public class ConcurrencyLimitServletFilterTest {
 
-    @Mock
-    Limiter<HttpServletRequest> limiter;
+    Predicate<HttpServletRequest> bypassActuatorEndpointPredicate = (request) ->
+            request.getRequestURI().contains("/admin");
```

Comment:
We should add a helper to the ServletLimiterBuilder to make this more ergonomic, similar to the existing partition helpers.